concepts/Common/Licensing/40 Troubleshooting.md

```diff
@@ -0,0 +1,5 @@
+If you have problems verifying your license key, try downloading it again in [Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/) and specifying it in your application config.
+
+If you have licensing-related questions or need help with a purchase, please email `clientservices@devexpress.com`. We will be happy to follow-up.
+
+If you have a valid license and face any problems with its verification please submit a support ticket via the [DevExpress Support Center](https://supportcenter.devexpress.com/ticket/create).
```

Comment:
Let's warn users that a support ticket should be a **private** and also ask to attach their license key to this ticket